docs/api.md

```diff
@@ -1046,20 +1047,24 @@ Advanced parameters:
 #### Request
 
 ```shell
-curl http://localhost:11434/api/embeddings -d '{
+curl http://localhost:11434/api/embed -d '{
```

Comment:
Let's add another `Request (Batch Embeddings)` example with this, and a simpler single string example as `Request`